.github/workflows/posix.yml

```diff
@@ -68,17 +68,14 @@ jobs:
         python-version: 3.7
     - name: Set extra env
       run: |
-        if [ "macos-11" == "${{ matrix.os }}" ]; then
-          echo "TRAVIS_OS_NAME=osx" >> $GITHUB_ENV;
-          echo "LDFLAGS=-L/Library/Developer/CommandLineTools/SDKs/MacOSX12.1.sdk/usr/lib" >> $GITHUB_ENV;
-          echo "LIBRARY_PATH=-L/Library/Developer/CommandLineTools/SDKs/MacOSX12.1.sdk/usr/lib" >> $GITHUB_ENV;
-        else
-          echo "TRAVIS_OS_NAME=${{ matrix.os }}" >> $GITHUB_ENV;
-        fi
+        # if [ "macos-11" == "${{ matrix.os }}" ]; then
+        #   echo "LDFLAGS=-L/Library/Developer/CommandLineTools/SDKs/MacOSX12.1.sdk/usr/lib" >> $GITHUB_ENV;
+        #   echo "LIBRARY_PATH=-L/Library/Developer/CommandLineTools/SDKs/MacOSX12.1.sdk/usr/lib" >> $GITHUB_ENV;
```

Comment:
I'll note that if this is needed again in the future or changes, there's a better way than hardcoding it. The path can be obtained from `xcrun --sdk macosx --show-sdk-path`. I just managed to get rid of this hardcoding in numpy CI.

Reply:
I added a comment in the commit to add the BLAS_SYMBOL_PREFIX definition to the pkgconfig file